From e28e63a2724896ab867302fdae911e186d623346 Mon Sep 17 00:00:00 2001
From: Jurgen Kramer <gtmkramer@xs4all.nl>
Date: Sun, 17 Dec 2017 08:48:46 +0100
Subject: [PATCH] ALSA: usb: native DSD support for Mytek Manhattan

This patch adds native DSD support for the Mytek Manhattan.

Signed-off-by: Jurgen Kramer <gtmkramer@xs4all.nl>
---
 sound/usb/quirks.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/sound/usb/quirks.c b/sound/usb/quirks.c
index a66ef57..3106577 100644
--- a/sound/usb/quirks.c
+++ b/sound/usb/quirks.c
@@ -1349,6 +1349,7 @@ u64 snd_usb_interface_dsd_format_quirks(struct snd_usb_audio *chip,
 	case USB_ID(0x20b1, 0x2008): /* Matrix Audio X-Sabre */
 	case USB_ID(0x20b1, 0x300a): /* Matrix Audio Mini-i Pro */
 	case USB_ID(0x22d9, 0x0416): /* OPPO HA-1 */
+	case USB_ID(0x25ce, 0x0021): /* Mytek Manhattan */
 	case USB_ID(0x2772, 0x0230): /* Pro-Ject Pre Box S2 Digital */
 		if (fp->altsetting == 2)
 			return SNDRV_PCM_FMTBIT_DSD_U32_BE;
-- 
2.9.5

